spec/ParseGlobalConfig.spec.js

```diff
@@ -0,0 +1,49 @@
+// run test when changing related file using
+// $ TESTING=1 node_modules/jasmine/bin/jasmine.js spec/ParseGlobalConfig.spec.js
```

Comment:
I think one of our team members is in the middle of making some changes to our testing framework, so this comment is probably going to become out of date soon. Can you remove it? We'll keep instructions for how to run tests in `CONTRIBUTING.md`.